runner/app/main.py

```diff
@@ -7,10 +7,13 @@
 from fastapi.routing import APIRoute
 from app.utils.hardware import HardwareInfo
 from app.live.log import config_logging
+from prometheus_client import Counter, generate_latest, CONTENT_TYPE_LATEST
+from starlette.responses import Response
 
 config_logging()
 logger = logging.getLogger(__name__)
 
+VERSION = Counter('version', 'Runner version', ['app', 'version'])
```

Comment:
the other version metrics are `Gauge` type. i don't know if it'd conflict with `Counter` or not. https://github.com/livepeer/studio/blob/2bb3b0ab4bd015e3e561266072580192af25cca3/packages/api/src/index.ts#L129-L142

when querying, we simply use the promql expression:

```
sum by (app, version) version{}
```